File: app/src/main/java/com/chuongvd/app/signal/data/source/remote/datasource/ISignalRemoteDataSource.java
package com.chuongvd.app.signal.data.source.remote.datasource;

import com.chuongvd.app.signal.data.entity.SignalEntity;
import com.chuongvd.app.signal.data.entity.UserEntity;
import com.chuongvd.app.signal.data.source.remote.request.LoginRequest;
import io.reactivex.Observable;
import java.util.List;

public interface ISignalRemoteDataSource {
    Observable<List<SignalEntity>> getRemoteListSignal();
}

File: app/src/main/java/com/chuongvd/app/signal/data/source/remote/datasource/ISignalRemoteDataSource.kt
package com.chuongvd.app.signal.data.source.remote.datasource

import com.chuongvd.app.signal.data.entity.SignalEntity
import io.reactivex.Observable

interface ISignalRemoteDataSource {
  val remoteListSignal: Observable<List<SignalEntity>>
}
